data/json/professions.json

```diff
@@ -6265,5 +6305,605 @@
+      },
+      "male": [ "briefs", "dress_shoes", "socks" ],
+      "female": [ "panties", "bra", "heels" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "master_chef",
+    "name": "Master Chef",
+    "description": "You were a world-renowned chef managing the crew of a Michelin Star restaurant.  Unfortunately, now your customers would much rather take a bite of you instead of your signature foie gras.",
+    "points": 4,
+    "skills": [
+      { "level": 8, "name": "cooking" },
+      { "level": 4, "name": "speech" },
+      { "level": 3, "name": "melee" },
+      { "level": 3, "name": "cutting" }
+    ],
+    "proficiencies": [
+      "prof_food_prep",
+      "prof_knife_skills",
+      "prof_baking",
+      "prof_frying",
+      "prof_baking_desserts_1",
+      "prof_frying_dessert",
+      "prof_baking_bread",
+      "prof_baking_desserts_2",
+      "prof_preservation",
+      "prof_food_canning",
+      "prof_food_curing"
+    ],
+    "items": {
+      "both": {
+        "items": [
+          "pants_checkered",
+          "socks",
+          "dress_shoes",
+          "platinum_watch",
+          "mbag",
+          "knife_chef",
+          "knife_bread",
+          "knife_steak",
+          "knife_paring",
+          "knife_meat_cleaver",
+          "knife_vegetable_cleaver",
+          "knife_carving"
+        ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "knife_butcher", "container-item": "sheath" },
+          { "item": "jacket_chef", "variant": "white_jacket_chef" },
+          { "item": "apron_cotton", "variant": "generic_apron_cotton" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "arsonist",
+    "name": "Arsonist",
+    "description": "You live to see the world burn, literally and figuratively.  Now that everyone has turned into a zombie, they're not going to miss their houses going up in flames.",
+    "points": 3,
+    "traits": [ "PYROMANIA" ],
+    "skills": [
+      { "level": 4, "name": "fabrication" },
+      { "level": 4, "name": "chemistry" },
+      { "level": 3, "name": "firstaid" },
+      { "level": 3, "name": "throw" },
+      { "level": 2, "name": "swimming" }
+    ],
+    "proficiencies": [ "prof_burn_care" ],
+    "items": {
+      "both": {
+        "items": [ "tshirt", "mbag", "socks", "sneakers", "pants_cargo" ],
+        "entries": [
+          { "group": "charged_smart_phone" },
+          { "item": "gasoline", "charges": 10000, "container-item": "jerrycan", "custom-flags": [ "auto_wield" ] },
+          { "item": "mask_filter", "ammo-item": "gasfilter_sm", "charges": 100 },
+          { "item": "ref_lighter", "charges": 50 },
+          { "item": "molotov", "count": 5 }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "panties", "bra" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "bomber",
+    "name": "Bomb Maker",
```

Comment:
Again, this doesn't seem like a profession to me. Have you met many professional bomb makers?